src/middlewares/commentMiddleware.js

```diff
@@ -59,7 +59,13 @@ export default class CommentMiddleware {
         manager: { id: managerId },
         requester: { id: requesterId }
       } = request.get({ plain: true });
-      if ([managerId, requesterId].includes(userId)) return next();
+      if ([managerId, requesterId].includes(userId)) {
```

Comment:
Please explain what's happening from the if statement to next()

Reply:
cool. So you know how it has to be the person who's on the other end of a travel request that receives a notification in the event that someone makes a comment, that's exactly what's happening here. First, we check if the logged-in user is the requestor or the manager of that travel request using `if ([managerId, requesterId].includes(userId))` , next we try to figure out whether this user who is trying to make a comment is a manager or a requestor `managerId === userId
          ? { userId: requesterId, isManager: true }
          : { userId: managerId, isManager: false };`
We then update the req.notify property with the appropriate data. In the commentService class, We simply send a warm notification to the right person after the comment has been recorded in the database

Reply:
Elaborate, Thanks. So if a manager makes a comment, the requester is notified and if a requester makes a comment the manager is notified, correct?

Reply:
@Baystef  Exactly bro. You are welcome. I also changed the route path for `get, put and update` travel requests. It wasn't properly named and it wasn't documented in the right manner. I wanted to be sure we are conforming to the RESTFUL API best practices. You can check for the changes in  the `user` routes file.

Reply:
Yeah, I noticed your changes, I'll go check properly. Weldone